salt/modules/debian_ip.py

```diff
@@ -1649,8 +1649,8 @@ def build_bond(iface, **settings):
     # Load kernel module
     __salt__["kmod.load"]("bonding")
 
-    # install ifenslave-2.6
```

Comment:
is the packagename -2.6 on older platforms?

Reply:
It is available on Debian 10 and 9, Ubuntu 18.04 and 20.04 as well as the **_ifenslave_** package.  Same versions etc

```
root@tdeb10:/home/david# lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
root@tdeb10:/home/david# apt-cache policy ifenslave*
ifenslave-2.6:
  Installed: (none)
  Candidate: 2.9
  Version table:
     2.9 500
        500 http://deb.debian.org/debian buster/main amd64 Packages
ifenslave:
  Installed: (none)
  Candidate: 2.9
  Version table:
     2.9 500
        500 http://deb.debian.org/debian buster/main amd64 Packages
root@tdeb10:/home/david# 
```

The ifenslave-2.6 package is removed on Debian 11 and Ubuntu 21.10 (hence will be gone in the upcoming 22.04)